lifenet/tasks.py

```diff
@@ -0,0 +1,96 @@
+from celery.decorators import periodic_task
+from celery.task.schedules import crontab
+from django.conf import settings
+from django_redis import get_redis_connection
+from temba_client.v2 import TembaClient
+
+from healthcheck import utils
+from lifenet.models import LNCheck
+from userprofile.models import HealthCheckUserProfile
+
+
+@periodic_task(run_every=crontab(minute="*/5"))
+def perform_sync_to_rapidpro():
+    r = get_redis_connection()
+    if r.get("perform_sync_to_rapidpro"):
+        return
+
+    with r.lock("perform_sync_to_rapidpro", 1800):
+        if (
+            settings.RAPIDPRO_URL
+            and settings.RAPIDPRO_TOKEN
+            and settings.RAPIDPRO_LIFENET_FLOW
```

Comment:
If lifenet is in the same org as tbconnect, then this will be fine. But if not, then you'll need a different `RAPIDPRO_TOKEN`, and if it's on a different instance, then you'll need a new `RAPIDPRO_URL`